### ISSUE TITLE
Windows minion user.rename gives exception

### ISSUE THREAD

**Issue by ssgward** [saltstack/salt#26788]
Here is the command run: ``salt Win2012 user.rename testuser01 testuser``

Here is the exception:
```
Win2012:
    The minion function caused an exception: Traceback (most recent call last):
      File "C:\salt\bin\lib\site-packages\salt\minion.py", line 1173, in _thread_return
        return_data = func(*args, **kwargs)
      File "C:\salt\bin\lib\site-packages\salt\modules\win_useradd.py", line 459, in rename
        if post_info['name'] != current_info['name']:
    KeyError: 'name'
```
Tested on Win 2012 with Win salt minion versions 2015.3, 2015.5 and 2015.8rc3 all with the same result.

Workaround (but may not be optimal) would be to delete the user and re-add with updated username.

**Comment by twangboy** [comment]
@ssgward Deleting the user and recreating would change the GUID as well as lose any groups or file permissions the user may have. I'll take a look at why this is failing.

**Comment by twangboy** [comment]
Should be fixed by #26821
